packages/patternfly-4/react-core/src/components/DataList/DataListAction.tsx

```diff
@@ -1,8 +1,22 @@
 import * as React from 'react';
-import { css } from '@patternfly/react-styles';
+import { css, pickProperties } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/DataList/data-list';
 import { Omit } from '../../helpers/typeUtils';
 
+export const DataListActionVisibility = pickProperties(styles.modifiers, [
+  'hidden',
+  'hiddenOnSm',
+  'hiddenOnMd',
+  'hiddenOnLg',
+  'hiddenOnXl',
+  'hiddenOn2Xl',
```

Comment:
Just want to confirm this should be `[hidden/visible]On2Xl` and not `[hidden/visible]On2xl`? Not sure how numbers play with camel case.

Reply:
Hmm... I'm trying the `[hidden/visible]On2Xl` props and they don't seem to be working. Can someone confirm? @tlabaj or @christiemolloy. I'm probably doing it wrong :)

Reply:
I actually was wondering the same thing...

Reply:
`hiddenOn2Xl` is correct. It comes from our Javascript variable names being camelcased in react-styles based on the css variables.

Reply:
I also do not see the [hidden/visible]On2Xl being applied.

Reply:
For now I wrote a workaround for that. Please review and let me know what you think.

Thanks.